### PR TITLE
update system-variables.md: add `mpp_version` and `mpp_exchange_compression_mode`

### PR DESCRIPTION
**Pull request by solotzg**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -402,6 +402,29 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
...
+    - `NONE`：无数据压缩
+    - `FAST`：快速模式，整体性能较好，压缩比小于 `HIGH_COMPRESSION`
+    - `HIGH_COMPRESSION`：高压缩比模式
+
+### `mpp_version` <span class="version-mark">从 v6.6.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 默认值：`UNSPECIFIED`
+- 取值范围：`UNSPECIFIED, 0, 1`
```
Comment (TomShawn) [inline system-variables.md:422]:
```suggestion
- 可选值：`UNSPECIFIED`，`0`，`1`
```

File: system-variables.md [hunk 2]
```diff
@@ -402,6 +402,29 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
 - 服务器端和客户端在一次传送数据包的过程中所允许最大的数据包大小，单位为字节。
 - 该变量的行为与 MySQL 兼容。
 
+### `mpp_exchange_compression_mode` <span class="version-mark">从 v6.6.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 默认值：`UNSPECIFIED`
+- 取值范围：`NONE, FAST, HIGH_COMPRESSION, UNSPECIFIED`
```
Comment (solotzg) [inline system-variables.md:410]:
```suggestion
- 可选值：`NONE, FAST, HIGH_COMPRESSION, UNSPECIFIED`
```

File: system-variables.md [hunk 3]
```diff
@@ -402,6 +402,29 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
...
+
+### `mpp_version` <span class="version-mark">从 v6.6.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 默认值：`UNSPECIFIED`
+- 可选值：`UNSPECIFIED`，`0`，`1`
+- 该变量用于选择不同版本的 MPP 执行计划
+    - `UNSPECIFIED`：自动选择最新版本，当前为 `1`
+    - `0`：兼容所有 TiDB 集群版本
```
Comment (joey-yez) [inline system-variables.md:425]: 此处是否需要说明一下，设定为 0 表示不支持压缩交换，即上面的参数不会生效

File: system-variables.md [hunk 4]
```diff
@@ -402,6 +402,29 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
 - 服务器端和客户端在一次传送数据包的过程中所允许最大的数据包大小，单位为字节。
 - 该变量的行为与 MySQL 兼容。
 
+### `mpp_exchange_compression_mode` <span class="version-mark">从 v6.6.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 默认值：`UNSPECIFIED`
+- 取值范围：`NONE, FAST, HIGH_COMPRESSION, UNSPECIFIED`
+- 该变量用于选择 MPP Exchange 算子的数据压缩模式
```
Comment (joey-yez) [inline system-variables.md:411]: 要说明一下：  mpp_version 为 1 时，该参数才会实际有作用。

File: system-variables.md [hunk 5]
```diff
@@ -402,6 +402,29 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
 - 服务器端和客户端在一次传送数据包的过程中所允许最大的数据包大小，单位为字节。
 - 该变量的行为与 MySQL 兼容。
 
+### `mpp_exchange_compression_mode` <span class="version-mark">从 v6.6.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 默认值：`UNSPECIFIED`
+- 可选值：`NONE, FAST, HIGH_COMPRESSION, UNSPECIFIED`
+- 该变量用于选择 MPP Exchange 算子的数据压缩模式
```
Comment (solotzg) [inline system-variables.md:411]:
```suggestion
- 该变量用于选择 MPP Exchange 算子的数据压缩模式，当 TiDB 选择版本号为 `1` 的 MPP 执行计划时生效。
```

File: system-variables.md [hunk 6]
```diff
@@ -402,6 +402,29 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
...
+
+### `mpp_version` <span class="version-mark">从 v6.6.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 默认值：`UNSPECIFIED`
+- 可选值：`UNSPECIFIED`，`0`，`1`
+- 该变量用于选择不同版本的 MPP 执行计划
+    - `UNSPECIFIED`：自动选择最新版本，当前为 `1`
+    - `0`：兼容所有 TiDB 集群版本
```
Comment (solotzg) [inline system-variables.md:425]:
```suggestion
    - `0`：兼容所有 TiDB 集群版本，MPP 版本大于 `0` 的新特性均不会生效。
```

File: system-variables.md (hunk 4 above)
Comment (solotzg) [reply system-variables.md]: Done

File: system-variables.md (hunk 3 above)
Comment (solotzg) [reply system-variables.md]: Done

File: system-variables.md [hunk 7]
```diff
@@ -402,6 +402,29 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
...
+    - `FAST`：快速模式，整体性能较好，压缩比小于 `HIGH_COMPRESSION`
+    - `HIGH_COMPRESSION`：高压缩比模式
+
+### `mpp_version` <span class="version-mark">从 v6.6.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 默认值：`UNSPECIFIED`
+- 可选值：`UNSPECIFIED`，`0`，`1`
+- 该变量用于选择不同版本的 MPP 执行计划
```
Comment (TomShawn) [inline system-variables.md:423]:
```suggestion
- 该变量用于指定不同版本的 MPP 执行计划。指定后，TiDB 会选择指定版本的 MPP 执行计划。
```

File: system-variables.md [hunk 8]
```diff
@@ -402,6 +402,29 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
...
+    - `UNSPECIFIED`：自动选择最新版本，当前为 `1`
```
Comment (TomShawn) [inline system-variables.md:424]:
```suggestion
    - `UNSPECIFIED`：表示未指定，此时 TiDB 自动选择最新版本 `1`。
```

File: system-variables.md [hunk 9]
```diff
@@ -402,6 +402,29 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
...
+    - `0`：兼容所有 TiDB 集群版本，MPP 版本大于 `0` 的新特性均不会生效。
+    - `1`：从 v6.6.0 版本开始引入，新特性 `TiFlash 引擎支持带压缩的数据交换`
```
Comment (TomShawn) [inline system-variables.md:426]:
```suggestion
    - `1`：从 v6.6.0 版本开始引入，用于开启 TiFlash 带压缩的数据交换，详情参见 [MPP Version 和 Exchange 数据压缩](/explain-mpp.md#mpp-version-和-exchange-数据压缩)。
```

File: system-variables.md [hunk 10]
```diff
@@ -402,6 +402,29 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
 - 服务器端和客户端在一次传送数据包的过程中所允许最大的数据包大小，单位为字节。
 - 该变量的行为与 MySQL 兼容。
 
+### `mpp_exchange_compression_mode` <span class="version-mark">从 v6.6.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 默认值：`UNSPECIFIED`
+- 可选值：`NONE, FAST, HIGH_COMPRESSION, UNSPECIFIED`
+- 该变量用于选择 MPP Exchange 算子的数据压缩模式，当 TiDB 选择版本号为 `1` 的 MPP 执行计划时生效。
```
Comment (TomShawn) [inline system-variables.md:411]:
```suggestion
- 该变量用于选择 MPP Exchange 算子的数据压缩模式，当 TiDB 选择版本号为 `1` 的 MPP 执行计划时生效。该变量值的含义如下：
```

File: system-variables.md [hunk 11]
```diff
@@ -402,6 +402,29 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
...
+    - `UNSPECIFIED`：自动选择，当前为 `FAST`
```
Comment (TomShawn) [inline system-variables.md:412]:
```suggestion
    - `UNSPECIFIED`：表示未指定，TiDB 将自动选择压缩模式，当前 TiDB 自动选择 `FAST` 模式
```

File: system-variables.md [hunk 12]
```diff
@@ -402,6 +402,29 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
...
+    - `NONE`：无数据压缩
```
Comment (TomShawn) [inline system-variables.md:413]:
```suggestion
    - `NONE`：不使用数据压缩
```

File: explain-mpp.md [hunk 13]
```diff
@@ -165,6 +165,34 @@ EXPLAIN ANALYZE SELECT COUNT(*) FROM t1 GROUP BY id;
...
 
+## MPP Version 和 Exchange 数据压缩
+
+从 v6.6.0 开始，MPP 执行计划新增字段 `MppVersion` 和 `Compression`。
+
+- `MppVersion`：MPP 执行计划的版本号，可通过系统变量 [`mpp_version`](/system-variables.md#mpp_version-从-v660-版本开始引入) 设置。
+- `Compression`：`Exchange` 算子的数据压缩模式，可通过系统变量 [`mpp_exchange_compression_mode`](/system-variables.md#mpp_exchange_compression_mode-从-v660-版本开始引入) 设置。如果未启用数据压缩，则不显示该字段。
+
+```sql
+MySQL > explain select count(*) as count_order from lineitem group by l_returnflag, l_linestatus order by l_returnflag, l_linestatus;
```
Comment (TomShawn) [inline explain-mpp.md:176]:
```suggestion
mysql > EXPLAIN SELECT COUNT(*) AS count_order FROM lineitem GROUP BY l_returnflag, l_linestatus ORDER BY l_returnflag, l_linestatus;
```

File: system-variables.md [hunk 14]
```diff
@@ -402,6 +402,29 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
...
+    - `FAST`：快速模式，整体性能较好，压缩比小于 `HIGH_COMPRESSION`
+    - `HIGH_COMPRESSION`：高压缩比模式
+
+### `mpp_version` <span class="version-mark">从 v6.6.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 默认值：`UNSPECIFIED`
+- 可选值：`UNSPECIFIED`，`0`，`1`
+- 该变量用于指定不同版本的 MPP 执行计划。指定后，TiDB 会选择指定版本的 MPP 执行计划。
```
Comment (TomShawn) [inline system-variables.md:423]:
```suggestion
- 该变量用于指定不同版本的 MPP 执行计划。指定后，TiDB 会选择指定版本的 MPP 执行计划。该变量值含义如下：
```

File: system-variables.md [hunk 15]
```diff
@@ -402,6 +402,29 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
 - 服务器端和客户端在一次传送数据包的过程中所允许最大的数据包大小，单位为字节。
 - 该变量的行为与 MySQL 兼容。
 
+### `mpp_exchange_compression_mode` <span class="version-mark">从 v6.6.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 默认值：`UNSPECIFIED`
+- 可选值：`NONE, FAST, HIGH_COMPRESSION, UNSPECIFIED`
```
Comment (TomShawn) [inline system-variables.md:410]:
```suggestion
- 可选值：`NONE`，`FAST`，`HIGH_COMPRESSION`，`UNSPECIFIED`
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

**Review by joey-yez — APPROVED** [review]
LGTM

**Review by ran-huang — APPROVED** [review]

**Pull request merged**